### PR TITLE
feat(infiscal): Added infiscal

### LINKED ISSUE #1: Add infisical module
# Open Source Secret Management

All-in-one platform to securely manage application configuration and secrets across your team and infrastructure.

Link: https://infisical.com/

---

How to test:

1. Initialize a Docker swarm with docker swarm init unless a Docker swarm is already running.
2. In main.tf, include the Terraswarm configuration (with the local path).
3. Execute `terraform init` && `terraform apply`.
4. Check if the Docker service deployed without errors. 🙂
5. Check if the Infisical service is usable according to their docs.

### PR DESCRIPTION
**Pull request by tbcashy**
Resolves https://github.com/ehwplus/terraswarm/issues/1

### CODE REVIEW

**Comment by tbcashy** [comment]
@tafaust Ready for review. 

Additional vars: https://raw.githubusercontent.com/Infisical/infisical/main/.env.example

File: modules/infiscal/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<!-- BEGIN_TF_DOCS -->
```
Comment (tafaust) [inline modules/infiscal/README.md:5]: # infiscal

See https://raw.githubusercontent.com/Infisical/infisical/main/.env.example for additionally required environmental variables.

```suggestion
<!-- BEGIN_TF_DOCS -->
```

**Review by tafaust — CHANGES_REQUESTED** [review]
Please update README.md accordingly with your hint for the additionally required .env vars.

**Review by tafaust — APPROVED** [review]

**Comment by tafaust** [comment]
TYSM @tbcashy for your contribution!

**Pull request merged**